traffic_stops/settings/deploy.py

```diff
@@ -114,4 +114,4 @@
 
     # List of email addresses that receive the report of non-compliance of
     # traffic stop reporting.
-    COMPLIANCE_REPORT_LIST = ('Ianmance@southerncoalition.org',)
+    COMPLIANCE_REPORT_LIST = ('wcarpenter@forwardjustice.org',)
```

Comment:
For now, let's set this to a Caktus address so we can see what it actually produces